src/main/java/org/traccar/config/Keys.java

```diff
@@ -843,6 +843,14 @@ private Keys() {
             List.of(KeyType.CONFIG),
             "max-age=3600,public");
 
+     /**
+     * Set HttpOnly attribut to the session cookie.
+     */
+    public static final ConfigKey<Boolean> WEB_COOKIE_HTTP_ONLY = new BooleanConfigKey(
+        "web.cookieHttpOnly",
```

Comment:
Any reason this needs to be configurable?

Reply:
Potentially this could provide backwards compatibility with clients that need to access this cookie client-side. But I'm not aware if such an implementation makes sense.

Reply:
If it's not required currently by the official app, I suggest we remove the configuration option.

Reply:
I agree and have removed the parameter. The cookie is now always httpOnly.